subprojects/packagefiles/libscpi/libscpi/meson.build

```diff
@@ -0,0 +1,29 @@
+subdir('src')
+
+libscpi_lib = both_libraries('scpi',
+  libscpi_srcs,
+  include_directories : 'inc',
+  install : true,
+)
+
+libscpi_dep = declare_dependency(
+  link_with : libscpi_lib,
+  include_directories : 'inc',
+)
+
+# Make this library usable from the system's
+# package manager.
+subdir('inc/scpi')
+install_headers(libscpi_hdrs, subdir : 'scpi')
+
+pkg_mod = import('pkgconfig')
+pkg_mod.generate(
+  name : 'libscpi',
+  filebase : 'libscpi',
```

Comment:
`filebase` is not needed (same as `name`).

Reply:
ack